Header header

string type

geometry_msgs/PointStamped objection_point

std_msgs/Float64 distance